account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -417,7 +418,11 @@ def _onchange_manual_reconcile_reference(self):
 
     @api.onchange("manual_amount_in_currency")
     def _onchange_manual_amount_in_currency(self):
-        if self.manual_line_id.exists() and self.manual_line_id:
+        if (
+            self.manual_line_id.exists()
+            and self.manual_line_id
+            and self.manual_kind != "liquidity"
+        ):
```

Comment:
Here, we may have a better fix I don't know, but here it was it fixes : 

When you click on the statement line (the liquidity line), `_onchange_manual_reconcile_reference` is called (to fill all manual fields from what I understand) and then in cascasde `_onchange_manual_amount_in_currency` is called and it recompute the manual_amount.
So, if the rate of the statement line is different from the current odoo rate at this date, the amount in company currency changes... (and become inconsitent with the real account move line amount)

I don't see any case when it should recompute the amount of the liquidity line.